website/docs/_posts/2017-10-05-configuration-options.md

```diff
@@ -511,14 +511,16 @@ Set this to true if you wish to receive one PR for every separate major version
 
 ## node
 
-Configuration specific for all node.js version updates (e.g. `.travis.yml`).
+Configuration specific for all Node.js version updates (e.g. `engines` field in `package.json`).
 
 | name    | value                                         |
 | ------- | --------------------------------------------- |
 | type    | object                                        |
-| default | { groupName: 'node.js', lazyGrouping: false } |
+| default | { groupName: 'Node.js', lazyGrouping: false } |
```

Comment:
Sorry, messed this one up again, I can switch this to `node.js` to avoid the noise that will be caused by changing this option.

Reply:
No, this is fine - I switched it. `groupName` is converted to lower case for branch names, so it won't cause any disruption. This will only impact the PR titles

Reply:
Ah, okay.